R-package/README.md

```diff
@@ -495,3 +405,12 @@ Known Issues
 ------------
 
 For information about known issues with the R package, see the [R-package section of LightGBM's main FAQ page](https://lightgbm.readthedocs.io/en/latest/FAQ.html#r-package).
+
+Outdated Links
+--------------
+
+This section is used to redirect people following links that have been removed from the documentation above.
+
+### Build Precompiled Binaries of the CRAN Package
```

Comment:
This section was 
> mainly for maintainers

Maybe we can simply remove it?

Reply:
oh true, ok I'll just remove it 😀

Reply:
removed in https://github.com/microsoft/LightGBM/pull/3612/commits/1fea8a32acd3b877e06712988fbcad28977a0a65